components/match2/wikis/ageofempires/match_summary.lua

```diff
@@ -112,7 +113,10 @@ function CustomMatchSummary._createGame(game, props)
 				:css('flex-direction', 'column')
 				:css('overflow', 'hidden')
 			Array.forEach(
-				Array.sortBy(game.opponents[opponentId].players, Operator.property('index')),
+				Array.sortBy(
+					Array.filter(game.opponents[opponentId].players, Table.isNotEmpty),
+					Operator.property('index')
```

Comment:
Is the sort still needed?

Reply:
Yes, display has to follow the input order on the match